webhook_server/app.py

```
@@ -194,62 +260,70 @@ async def process_webhook(request: Request, background_tasks: BackgroundTasks) -
+        Note:
+            All exceptions are caught and logged but never propagated, since this
+            runs asynchronously after the HTTP response is sent.
+        """
+        try:
+            # Initialize GithubWebhook inside background task to avoid blocking webhook response
+            _api: GithubWebhook = GithubWebhook(hook_data=_hook_data, headers=_headers, logger=_logger)
+            await _api.process()
+            _logger.success(f"{log_context} Webhook processing completed successfully")  # type: ignore
+        except RepositoryNotFoundInConfigError:
+            # Repository-specific error - not exceptional, log as error not exception
+            _logger.error(f"{log_context} Repository not found in configuration")
```

Ruff (0.14.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)